src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs

```diff
@@ -57,8 +55,9 @@ internal class JwtSecurityTokenHandlerSignalR : JwtSecurityTokenHandler
             JwtPayload payload = new JwtPayload(issuer, audience, (subject == null ? null : OutboundClaimTypeTransform(subject.Claims)), notBefore, expires, issuedAt);
             JwtHeader header = new JwtHeader(kid, algorithm);
 
-            if (subject?.Actor != null)
-                payload.AddClaim(new Claim(JwtRegisteredClaimNames.Actort, CreateActorValue(subject.Actor)));
+            // Because subject.Actor is always null, we skip this
```

Comment:
remove code, and merge the comment with the class comment above

Reply:
fixed